CollAction/CollAction.csproj

```diff
@@ -4,15 +4,21 @@
     <TypeScriptCompileBlocked>true</TypeScriptCompileBlocked>
     <UserSecretsId>aspnet-CollAction-29a5d7b8-3530-4482-9ed6-572bd6c178fa</UserSecretsId>
     <DockerComposeProjectPath>..\docker-compose.dcproj</DockerComposeProjectPath>
-    <TypeScriptToolsVersion>3.4.2</TypeScriptToolsVersion>
+    <TypeScriptToolsVersion>3.3</TypeScriptToolsVersion>
```

Comment:
Moving down on the typescript version?

Reply:
This is the "typescript tools" version, for visual studio. It only goes up to 3.3, I updated it earlier for the donations page in the other pull request, but it only gave me warnings.